src/components/help/help.pw.tsx

```diff
@@ -0,0 +1,392 @@
+    );
+    await expect(helpComponent(page).locator("span")).toHaveCSS(
+      "outline",
+      "rgba(0, 0, 0, 0) solid 3px"
+    );
+  });
+
+  test("should have the expected styling when the focusRedesignOptOut is true", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(<HelpComponentTest />, {
+      hooksConfig: { focusRedesignOptOut: true },
+    });
+
+    const elementLocator = helpComponent(page);
+    await elementLocator.focus();
+    await expect(helpComponent(page).locator("span")).toHaveCSS(
+      "outline",
+      "rgb(255, 188, 25) solid 2px"
+    );
+  });
+});
+
+test.describe("Testing Help component properties", () => {
+  testData.forEach((ariaLabel) => {
+    test(`should check ariaLabel as ${ariaLabel}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest ariaLabel={ariaLabel} />);
+
+      await expect(helpComponent(page)).toHaveAttribute(
+        "aria-label",
+        ariaLabel
+      );
+    });
+  });
+
+  testData.forEach((className) => {
+    test(`should check className as ${className}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest className={className} />);
+
+      await containsClass(helpComponent(page), className);
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check children as ${children}`, async ({ mount, page }) => {
+      await mount(<Help> {children} </Help>);
+
+      await helpComponent(page).hover();
+      const tooltip = getDataElementByValue(page, "tooltip");
+      await expect(tooltip).toBeVisible();
+      await expect(tooltip).toHaveText(children);
+    });
+  });
+
+  testData.forEach((helpId) => {
+    test(`should check id as ${helpId}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest helpId={helpId} />);
+
+      await expect(helpComponent(page)).toHaveId(helpId);
+    });
+  });
+
+  ["https://carbon.sage.com", "https://www.google.com"].forEach((hrefLink) => {
+    test(`should check href link as ${hrefLink}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest href={hrefLink} />);
+
+      await expect(helpComponent(page)).toHaveAttribute("href", hrefLink);
+    });
+  });
+
+  [true, false].forEach((boolVal) => {
+    test(`should check when isFocused prop is passed as ${boolVal}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest isFocused={boolVal}>{tooltipText}</HelpComponentTest>
+      );
+
+      const tooltip = getDataElementByValue(page, "tooltip");
+      if (boolVal === true) {
+        await expect(tooltip).toBeVisible();
+      } else {
+        await expect(tooltip).not.toBeVisible();
+      }
+    });
+  });
+
+  [-1, 0, 1].forEach((tabIndex) => {
+    test(`should check tabIndex as ${tabIndex}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest tabIndex={tabIndex} />);
+
+      if (tabIndex === -1) {
+        await page.keyboard.press("Tab");
```

Comment:
**suggestion(non-blocking):** You should be able to move this tab presses and the one on line 136 outside of the `if` statement. As whether the `tabIndex` is `-1`, `0` or `1` you still need to press the tab key.